config/routes.rb

```diff
@@ -309,4 +309,6 @@
 
   match ':controller(/:action(/:id))(.:format)'
 
+  match 'gridsEmbed/:tagname' => 'notes#embed
```

Comment:
Ah - take a look, there's a missing end quote here. That's why the tests are failing. But this is looking good!